### PR TITLE
[Meetup] Fixes Issue #3005 Descriptors for Week of Month not matching

### LINKED ISSUE #3005: [Meetup] Descriptors for Week of Month  in Problem Statement Don't Match the Descriptors in the  Tests
The descriptors for the week in the instructions.md do not match what is used in the unit test.  The best fix in my opinion would be to modify the instructions.md to match the unit test.  This would maintain the validity of existing submissions.

instructions.md:
    first, second, third, fourth, fifth

meetup_test.py
    1st, 2nd, 3rd, 4th, 5th

### PR DESCRIPTION
**Pull request by ErnieWhite**
Fixes #3005
 
Modified the necessary files and regenerated the test files.  Tested against the new example.py,  passing all 103 test.

### CODE REVIEW

**Comment by BethanyG** [comment]
@ErnieWhite - _this is awesome!_  Thanks so much for submitting this PR.  One quick note before I formally review:  you accidentally checked in your local `.gitignore` file.  That should be removed before we merge this.

**Comment by BethanyG** [comment]
Additionally, we have 63 of the 100+ tests for the exercise failing in the CI.  Investigating......

**Comment by ErnieWhite** [comment]
I am an idiot.  I forgot to commit the change from set to tuple in example.py

**Comment by BethanyG** [comment]
> I am an idiot. I forgot to commit the change from set to tuple in example.py

_You are in good company._  I cannot tell you how many times I've gotten mad at the CI, only to discover that I've made a very similar mistake in something I am PR-ing.  I am the **queen** of the idiots. 😆

**Comment by BethanyG** [comment]
@ErnieWhite  - why did you close this PR?  Was there something wrong with it?

**Comment by ErnieWhite** [comment]
Did not realize I did.  I am trying to remove the .gitignore and fix the mistake I made.

**Comment by ErnieWhite** [comment]
I couldn't figure out how to remove it. So I deleted everything and did it over.  If you can just remove the file that would work, or I can make a new pull request that has working code and no .gitignore

**Comment by BethanyG** [comment]
I'll remove it (I think I can).  BRB.

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@ bin/configlet.exe
 .pytest_cache
 __pycache__
 .venv
+env/
```
Comment (BethanyG) [inline .gitignore:16]:
```suggestion
```

**Review by BethanyG — APPROVED** [review]
@ErnieWhite - This looks good!  Thanks for updating the template and regenerating the test files.  Its much appreciated (_it will be less appreciated by the people who now have to update their solutions ..._).  🌟

**Pull request merged**